gmso/tests/test_topology.py

```diff
@@ -1010,3 +1011,7 @@ def test_units(self, typed_ethane):
             1.60217662e-19 * u.Coulomb,
             0.1 * u.Unit("test_charge", registry=reg.reg),
         )
+
+    def test_write_forcefield(self, typed_benzene_aa_system):
```

Comment:
I think you should merge the two `test_write_forcefield` functions into one.

Reply:
Good catch, I didn't notice that. Should be done now.